### PR TITLE
Plain-text mail templates

### PR DESCRIPTION
**Pull request by patrickvenetz**
This Pull Request adds plain-text mail templates.

To help handling changes in HTML templates, a script will help you convert these versions in to plain-text ones and you may alter them thereafter:

```sh
./packages/mail/script/generateTextTemplates.js
```

### CODE REVIEW

File: packages/mail/lib/sendMailTemplate.js [hunk 1]
```diff
@@ -27,16 +27,19 @@ const {
...
+    debug(`template "${name}" not found in templates folder`, { htmlPath })
     return false
   }
 
-  const contents = fs.readFileSync(templatePath, 'utf8')
-  return contents
+  return {
+    html: fs.readFileSync(htmlPath, 'utf8'),
+    text: (fs.existsSync(textPath) && fs.readFileSync(textPath, 'utf8')) || null
+  }
```
Comment (patte) [inline packages/mail/lib/sendMailTemplate.js]: two IO tasks could be parallelized

File: packages/mail/script/generateTextTemplates.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const htmlToText = require('html-to-text')
+const path = require('path')
+const Promise = require('bluebird')
+const recursive = require('recursive-readdir')
+
+const run = async () => {
+  const files = await recursive(
+    path.resolve(`${__dirname}/../templates`),
+    [file => path.extname(file) !== '.html']
+  )
```
Comment (patte) [inline packages/mail/script/generateTextTemplates.js]: add an `argv` option to only select one template

File: packages/mail/lib/sendMailTemplate.js [hunk 3]
```diff
@@ -210,7 +213,7 @@ module.exports = async (mail, context, log) => {
     subject: mail.subject,
     from_email: mail.fromEmail || DEFAULT_MAIL_FROM_ADDRESS,
     from_name: mail.fromName || DEFAULT_MAIL_FROM_NAME,
-    html: getTemplate(mail.templateName),
+    ...getTemplates(mail.templateName),
     merge_language: mail.mergeLanguage || 'handlebars',
     global_merge_vars: mergeVars,
     auto_text: true,
```
Comment (patte) [inline packages/mail/lib/sendMailTemplate.js]: set `auto_text` to false if the text template is available

**Comment by patrickvenetz** [comment]
Thus far ASCII beautification applied on `signin.txt` and `signin_code.txt` (as few as possible).

File: packages/mail/lib/sendMailTemplate.js (hunk 1 above)
Comment (patrickvenetz) [reply packages/mail/lib/sendMailTemplate.js]: Done.

File: packages/mail/script/generateTextTemplates.js (hunk 2 above)
Comment (patrickvenetz) [reply packages/mail/script/generateTextTemplates.js]: Done.

File: packages/mail/lib/sendMailTemplate.js (hunk 3 above)
Comment (patrickvenetz) [reply packages/mail/lib/sendMailTemplate.js]: Done, too.

**Pull request merged**